<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>449</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>553</width>
    <height>449</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>553</width>
    <height>449</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alert'Me</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ressources.qrc">
    <normaloff>:/new/prefix1/Images/icon.png</normaloff>:/new/prefix1/Images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Bt_close_ui">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermer la fenêtre</string>
    </property>
    <property name="icon">
     <iconset resource="Ressources.qrc">
      <normaloff>:/new/prefix1/Images/exit.png</normaloff>:/new/prefix1/Images/exit.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="Pb_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>531</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="Ressources.qrc">
       <normaloff>:/new/prefix1/Images/fire.png</normaloff>:/new/prefix1/Images/fire.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Message global</string>
     </attribute>
     <widget class="QPushButton" name="Mybt_intrusion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>507</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer message intrusion</string>
      </property>
      <property name="icon">
       <iconset resource="Ressources.qrc">
        <normaloff>:/new/prefix1/Images/intrusion.png</normaloff>:/new/prefix1/Images/intrusion.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Mybt_incendie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>507</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer message incendie</string>
      </property>
      <property name="icon">
       <iconset resource="Ressources.qrc">
        <normaloff>:/new/prefix1/Images/fire.png</normaloff>:/new/prefix1/Images/fire.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="Check_notify_sound">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>511</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Notification sonore sur les ordinateurs</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="txttosend_free">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>507</width>
        <height>101</height>
       </rect>
      </property>
      <property name="markdown">
       <string>Saisissez le texte de votre message ...

</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Condensed,'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:7px; margin-bottom:7px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Saisissez le texte de votre message ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mybt_free">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>507</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer message libre</string>
      </property>
      <property name="icon">
       <iconset resource="Ressources.qrc">
        <normaloff>:/new/prefix1/Images/send.png</normaloff>:/new/prefix1/Images/send.png</iconset>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>507</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="Ressources.qrc">
       <normaloff>:/new/prefix1/Images/intrusion.png</normaloff>:/new/prefix1/Images/intrusion.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Message individuel</string>
     </attribute>
     <widget class="QTextEdit" name="txttosend">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>60</y>
        <width>514</width>
        <height>221</height>
       </rect>
      </property>
      <property name="markdown">
       <string>Saisissez le texte de votre message ...

</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Condensed,'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:7px; margin-bottom:7px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Saisissez le texte de votre message ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mybt_admin">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>288</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer message</string>
      </property>
      <property name="icon">
       <iconset resource="Ressources.qrc">
        <normaloff>:/new/prefix1/Images/send.png</normaloff>:/new/prefix1/Images/send.png</iconset>
      </property>
     </widget>
     <widget class="QComboBox" name="My_Combo_IP">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>311</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>19</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Choisissez votre destinataire</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset resource="Ressources.qrc">
       <normaloff>:/new/prefix1/Images/srv_ok.png</normaloff>:/new/prefix1/Images/srv_ok.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Logs</string>
     </attribute>
     <widget class="QPlainTextEdit" name="My_logs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>507</width>
        <height>241</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Port TCP d'écoute :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fichier table destinataires :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_tcp_port">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_arp_ok">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ip_listen">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP d'écoute :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset resource="Ressources.qrc">
       <normaloff>:/new/prefix1/Images/configuration.png</normaloff>:/new/prefix1/Images/configuration.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QCheckBox" name="Check_open_start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Ouvrir la fenêtre de l'application au démarrage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Mybt_saveconfig">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>507</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer la configuration</string>
      </property>
      <property name="icon">
       <iconset resource="Ressources.qrc">
        <normaloff>:/new/prefix1/Images/save.png</normaloff>:/new/prefix1/Images/save.png</iconset>
      </property>
     </widget>
     <widget class="QCheckBox" name="Check_open_popup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>511</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Ouvrir les messages individuels sous forme de popup dans la zone de notification</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Icon_server_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>416</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Ressources.qrc">:/new/prefix1/Images/srv_ok.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_server_status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>416</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_admin">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>410</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Ressources.qrc">
      <normaloff>:/new/prefix1/Images/lock.png</normaloff>:/new/prefix1/Images/lock.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
